consensus/ibft/signer/helper_test.go

```diff
@@ -73,7 +73,7 @@ func Test_wrapCommitHash(t *testing.T) {
 	assert.Equal(t, expectedOutput, output)
 }
 
-// nolint
+//nolint
```

Comment:
Maybe revert this unnecessary change?

Reply:
It is not unnecessary, it fails because of that